Client/Frontend/Home/TopSitesPanel.swift

```diff
@@ -10,6 +10,7 @@ import Storage
 private let log = Logger.browserLogger
 
 private let ThumbnailIdentifier = "Thumbnail"
+private let FrecencyLimit = 20
```

Comment:
The issue with calculating it before hand is that the height/width of the collection view change when the device is rotated. I actually don't see the harm in setting the limit to 100 or something large since the data source's numberOfItemsInSection takes the minimum of the cursor result count and thumbnailCount https://github.com/mozilla/firefox-ios/blob/master/Client/Frontend/Home/TopSitesPanel.swift#L344